app/boards/shields/redox/redox.zmk.yml

```
@@ -0,0 +1,11 @@
+file_format: "1"
+id: redox
+name: Redox
+type: shield
+url: https://github.com/mattdibi/redox-keyboard
+requires: [pro_micro]
+features:
+  - keys
```

This has underglow, right?
```suggestion
  - keys
  - underglow
```

---

correct, added